fabtests/multinode/src/harness.c

```diff
@@ -45,6 +45,18 @@
 #include <core.h>
 struct pm_job_info pm_job;
 
+static int parse_caps(char* caps)
+{
+	if (strcmp(caps, "msg") == 0) {
+		return multi_msg;
+	} else if (strcmp(caps, "rma") == 0) {
+		return multi_rma;
+	} else {
+		printf("Warn: Invalid capability, defaulting to msg\n");
+		return multi_msg;
```

Comment:
Abort if the capability is wrong.  Don't change the value, so that incorrect tests scripts fail, rather than succeed with a warning buried in the output.

Reply:
*bump* previous comment.